code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -0,0 +1,107 @@
+/obj/item/living_heart
+	name = "Living Heart"
+	desc = "Link to the worlds beyond."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "living_heart"
+	///Target
+	var/datum/mind/target
```

Comment:
Remove the reference to the mind on destruction and handle it being destroyed before the item is.

Reply:
what?